.env.template

```diff
@@ -3,5 +3,9 @@ ENVIRONMENT=local
 APP_NAME=getgather-local
 LOG_LEVEL=DEBUG
 
+# Authentication credentials
+CNN_USERNAME=
+CNN_PASSWORD=
```

Comment:
This `.env` is typically for running the server and not for testing, so it's better to leave out env for tests (like these CNN credentials) as they can be supplied via other means (e.g. `export`).